### PR TITLE
generic.test: skip cluster leak warning for lvm

### PR DESCRIPTION
**Pull request by XueqiangWei**
Signed-off-by: Xueqiang Wei <xuwei@redhat.com>
ID: 1654097

### CODE REVIEW

**Comment by XueqiangWei** [comment]
(1/4) repeat1.Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.lvm.lvm_create: PASS (80.92 s)
 (2/4) repeat1.Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.lvm.lvm_fill: PASS (149.64 s)
 (3/4) repeat1.Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.lvm.lvm_ioquit: PASS (127.39 s)
 (4/4) repeat1.Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.lvm.lvm_clean: PASS (62.62 s)

Tested with python3, also passed.

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

**Comment by XueqiangWei** [comment]
@YongxueHong @luckyh  Please help review it. Many thanks.

File: generic/tests/cfg/lvm.cfg [hunk 1]
```diff
@@ -17,6 +17,7 @@
     post_command_noncritical = no
     clean = no
     check_mount = "mountpoint /mnt/kvm_test_lvm"
+    skip_cluster_leak_warn = yes
```
Comment (luckyh) [inline generic/tests/cfg/lvm.cfg]: Should only apply for `lvm_ioquit`, right?

Comment (XueqiangWei) [reply generic/tests/cfg/lvm.cfg]: No, lvm_clean also needed.

**Comment by CongLi** [comment]
Ack.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**